CHANGELOG.md

```diff
@@ -48,6 +48,12 @@ _None._
 
 _None._
 
+## 1.13.1
```

Comment:
Nit: Last time I checked, any fixes or updates to libraries should be set with a beta version, and release managers will take care of releasing a new version. So here we should use `1.13.1-beta-1` instead.

Reply:
FWIW, there is no need to update the version header in the `CHANGELOG` while in development. The release manager that will ship the next stable version will do that.

This is advantageous because it allows all different kind of changes to land in the main branch and for the semantic version value to be evaluated only at the time of cutting a new stable version.

Example: If only bug fixes landed, we shall make the new version a patch. But if a new feature landed as well as bug fixes, it should be a new minor version.